src/librustc_middle/ty/list.rs

```diff
@@ -35,6 +35,22 @@ pub struct List<T> {
     opaque: OpaqueListContents,
 }
 
+unsafe impl<'a, T: 'a> rustc_data_structures::tagged_ptr::Pointer for &'a List<T> {
+    const BITS: usize = std::mem::align_of::<usize>().trailing_zeros() as usize;
```

Comment:
This should be `align_of::<List<T>>`, no?

Reply:
`List<T>` is unsized as mentioned elsewhere so we can't get the alignment directly. It's `repr(C)` though so the first field is the alignment of the type.